lib/cli/line-processor.js

```diff
@@ -75,13 +75,13 @@ module.exports = class LineProcessor {
       callback(reportMissingArgument('Host'));
       return;
     }
-    const args = utils.split(tokens, ' ', 2);
-    let [scheme, authority] = utils.split(args[0], '://', 1, true);
+    const args = autocomplete.split(tokens, ' ', 2);
```

Comment:
I agree, but see https://github.com/metarhia/jstp/pull/246/files#r132857348 We must arrange such common utilities to have code consistency over project, Meterhia stack and JS stories in general.